test/helpers/kubectl.go

```diff
@@ -1093,7 +1093,19 @@ func (kub *Kubectl) GetNodeIPByLabel(label string, external bool) (string, error
 		return "", fmt.Errorf("no matching node to read IP with label '%v'", label)
 	}
 
-	return out, nil
+	for _, ipStr := range strings.Split(out, " ") {
```

Comment:
Nit:
```suggestion
	for _, ipStr := range strings.Fields(out) {
```

This will eat all whitespaces (also multiple occurences) and would make this a bit more resilient against potential command output format changes.